numpy/random/bit_generator.pyi

```diff
@@ -0,0 +1,61 @@
+from threading import Lock
+from typing import Literal, Optional
+
+from numpy import DtypeLike, ndarray
+from numpy.typing import ArrayLike, _ShapeLike
+
+class SeedlessSeedSequence:
+    def generate_state(
+        self, n_words: int, dtype: DtypeLike = ...
+    ) -> ndarray: ...
+    def spawn(self, n_children: int) -> List[SeedlessSeedSequence]: ...
+
+class SeedSequence:
+    def __init__(
+        self,
+        entropy: Optional[int, Sequence[int]] = ...,
+        *,
+        spawn_key: Tuple[int, ...] = ...
+    ) -> None: ...
```

Comment:
`__init__` also has the `pool_size` and `n_children_spawned` keywords:
https://github.com/numpy/numpy/blob/871009f6415a2f588230e2e36bc9dbe751698ee1/numpy/random/bit_generator.pyx#L299-L300

Reply:
I think I deleted quite a bit by accident when I copied and pasted.